### PR TITLE
Prefer any lecture hall over Interims

### PR DESCRIPTION
**Pull request by joschahenningsen**
I'm kind of late to the party concerning #26 but I think this problem is not very hard to solve:

My code just checks whether a duplicate lecture also takes place in the Interims lecture hall and if so, takes the other location.

This would not help with the live-stream to the other campus though. That issue could be tackled by instead of checking for "Interims" in the location using the description and look for "Videoübertragung". Since almost all lectures are streamed right now I think that could lead to many false negatives currently.

**This is just a rough draft, lmk if there is anything that could still be improved :)**

### CODE REVIEW

File: src/handler.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public function cleanEvent(Event &$e) {
         $event->setLocation($location);
 
         //Remove the TAG and anything after e.g.: (IN0001) or [MA0001]
-        $summary = preg_replace('/([\(\[](?:(?:IN|MA|WI)\d+,?\s?)+[\)\]]).+/', '', $summary);
+        $summary = preg_replace('/([(\[](?:(?:IN|MA|WI)\d+,?\s?)+[)\]]).+/', '', $summary);
```
Comment (kordianbruck) [inline src/handler.php:34]: Whats the reason behind this change?

**Review by kordianbruck — APPROVED** [review]
This actually looks good. Github has a "draft" feature, so I marked it as such. No real comments here, so feel free to send it over once you feel more confident. Any chance you tested this locally?

File: src/handler.php (hunk 1 above)
Comment (joschahenningsen) [reply src/handler.php:34]: I do as Intellij guides :smile: 

It was marked as redundant, so I figured, why not update this as well?

Comment (kordianbruck) [reply src/handler.php:34]: Fine by me :)

**Comment by joschahenningsen** [comment]
> Any chance you tested this locally?

Sure, check out https://proxy.joschas.page, for my usecase it worked fine.

**Comment by kordianbruck** [comment]
Oh cool, love that improved design - any chance you want to share that as well? 

The tests seem to be failing, if you are super motivated you could check why the import doesn't go through.

**Comment by joschahenningsen** [comment]
I'd love to do both right after I got some sleep! :D

**Comment by kordianbruck** [comment]
This looks good to go for me.

**Pull request merged**